### PR TITLE
Add note that the integration is broken and not likely to be reliable

### PR DESCRIPTION
**Pull request by lewisdiamond**
See https://github.com/home-assistant/core/issues/101912

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/myq.markdown [hunk 1]
```diff
@@ -20,6 +20,7 @@ ha_platforms:
 ha_dhcp: true
 ha_integration_type: integration
 ---
+## This integration is broken and unlikely to ever be reliable. Chamberlain / LiftMaster / MyQ is hostile to 3rd party use of their API. Consider using a different brand if you want a garage door opener that works with HomeAssistant.
```
Comment (lewisdiamond) [inline source/_integrations/myq.markdown]:
```suggestion
## This integration is broken and unlikely to ever be reliable. Chamberlain / LiftMaster / MyQ is hostile to 3rd party use of their API. Consider using a different brand if you want a garage door opener that works with Home Assistant.
```

File: source/_integrations/myq.markdown [hunk 2]
```diff
@@ -20,6 +20,7 @@ ha_platforms:
 ha_dhcp: true
 ha_integration_type: integration
 ---
+## This integration is broken and unlikely to ever be reliable. Chamberlain / LiftMaster / MyQ is hostile to 3rd party use of their API. Consider using a different brand if you want a garage door opener that works with Home Assistant.
```
Comment (frenck) [inline source/_integrations/myq.markdown:23]: There are better places for these things. We have alerts available if that is needed and are considering at this point.

Please track the issue tracker for status updates.

Thanks! 👍 

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/myq.markdown (hunk 2 above)
Comment (lewisdiamond) [reply source/_integrations/myq.markdown:23]: The availability of the integration is misleading and will result in people buying a product that actually does not work with the expectation that it does work. A warning should be added for sure.

Comment (frenck) [reply source/_integrations/myq.markdown:23]: Thanks @lewisdiamond, however, see my previous answer, thanks 👍

**Pull request closed without merging**